charts/incubator/ts3-manager/values.yaml

```
@@ -0,0 +1,15 @@
+image:
+  repository: joni1802/ts3-manager
+  pullPolicy: IfNotPresent
+  tag: v2.2.0@sha256:9db01d161d2b37e8e872ed701cbedb3986eff46721d12f4d59279abfec5b0354
+
+env:
+  PORT: "{{ .Values.service.main.ports.main.port }}"
+
```

```suggestion
env:
  PORT: "{{ .Values.service.main.ports.main.port }}"

secretEnv:
  JWT_SECRET: "changeme"
```

Env variables appearing in `questions.yaml` should always be on `values.yaml` as well